apps/admin/frontend/src/components/reporting/tally_report_viewer.tsx

```diff
@@ -252,17 +252,23 @@ export function TallyReportViewer({
     );
 
     setProgressModalText('Printing Report');
+    const reportProperties = {
+      filter: JSON.stringify(filter),
+      groupBy: JSON.stringify(groupBy),
+    } as const;
```

Comment:
This is a brute force approach to adding information in logging, but I think it's valuable to include this information and, due to the infrequency with which logs will be looked at, not worth investing time in something prettier.